### ISSUE TITLE
Remove the templates_dir warning

### ISSUE THREAD

**Issue by BaptisteGelez** [Plume-org/Plume#244]
At each launch, Rocket tells us:

```
Error: missing configuration key: 'template_dir'
    => Warning: Using default templates directory '"/home/baptiste/Projects/plume/templates"'
```

And many people think it is an error, and that something went wrong during the installation. We should either set it during the setup with the `.env`, or use a custom Rocket config from Rust code.

Plume version (and/or commit): 0.2.0

**Comment by igalic** [comment]
It feels like we should make the `templates` (and static) dir its own repository so people can fork them to make their own themes

**Comment by BaptisteGelez** [comment]
Actually, I would like to use [askama](https://github.com/Plume-org/Plume/issues/161) for templates in the future, which would make it harder to put them in their own repository… However, theming with some CSS could be allowed in the future too.

Maybe we could still put them in another repository and "load" them with a git submodule (So if you want custom templates, you would clone Plume without the submodule, then clone in the `template` dir (or whatever) the repo of the theme we want…)

**Comment by igalic** [comment]
> Actually, I would like to use askama for templates in the future, which would make it harder to put them in their own repository… However, theming with some CSS could be allowed in the future too.

can you elaborate on *why* it would make it harder?

**Comment by BaptisteGelez** [comment]
Templates are embedded in the binary with askama, they are not read from the file system when the app starts. So if you want to change them you should recompile the whole app (well actually it's true that's it is not more complicated, only a bit longer).

**Comment by fdb-hiroshima** [comment]
Fixed by #327
